packages/terra-functional-testing/src/commands/viewport-helpers.js

```diff
@@ -0,0 +1,73 @@
+const Logger = require('../logger/logger');
+const terraViewports = require('../util/viewports');
```

Comment:
Nit: I'd probably import this as a constant. It doesn't really matter, just a little more standard.

```js
const { TERRA_VIEWPORTS } = require('../util/viewports'); // This directory/file name makes it sounds functional rather than static constants
```

Reply:
That makes sense. Updated and changed the directory name from util to static-assets. https://github.com/cerner/terra-toolkit/pull/438/commits/402ee523e9eabedfc323de69d40318f8b075dd7d